styleguide/source/_patterns/02-organisms/03-sections/06-audience-featured-hero/audience-featured-hero.twig

```diff
@@ -1,4 +1,4 @@
-<div class="layout_one_up audience_featured-hero">
+<div class="audience_featured-hero">
```

Comment:
Since replacing the `.layout` class is being done iteratively, I tried to maintain the current alignment of things where I could by using parts of the grid (for example, I didn't want to wrap entire Template patterns in a `.container` when only a handful of the included patterns knew how to be in a `.container`). This is what I meant by my blurb in the "Additional Notes" section. I couldn't think of a better solution. Sorry if that wasn't clear.

`.container-with-grid` was the direct replacement for `.layout` - we wanted to call it something more verbose to describe what it did. `.container-with-grid` is just `.container` and `.grid` rolled into one DOM level, so it made sense to use it in cases where adding a `.container` to the whole pattern wasn't appropriate.